administrator/components/com_admin/sql/updates/mysql/4.2.0-2022-06-16.sql

```diff
@@ -21,6 +21,13 @@ CREATE TABLE IF NOT EXISTS `#__guidedtours` (
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
 
+--
+-- Dumping data for table `#__guidedtours`
```

Comment:
Could you please increase the file version to 4.3?
Also check the other SQL what comment structure should be used.